### DISCUSSION TITLE
Support OSCAL format

### DISCUSSION THREAD

**Discussion by ab-smith** [intuitem/ciso-assistant-community#3361] [Ideas]
https://pages.nist.gov/OSCAL/

for input and export

**Comment by Wisw** [comment]
Any update or timeline that we are going to get this OSCAL integration feature any time soon for export and import to multiple formats.

Or is there any way we can communicate via APIs to this tool to POST data upon conversion via CLI tools available?

**Comment by Wisw** [comment]
@ab-smith any timeline to take this up?

**Comment by ab-smith** [comment]
Hello, expected for Q2.

**Reply by newsgrep** [reply]
@ab-smith @eric-intuitem May you give us a new ETA on this one.

**Comment by newsgrep** [comment]
Will this come with [OLIR](https://csrc.nist.gov/Projects/olir/derived-relationship-mapping#/) (csv/json) support? 
Will it allow to import xlsx documents structured like [sp800-53r5-to-iso-27001-mapping-2022-OLIR-2023-10-12-UPDATED.xlsx](https://csrc.nist.gov/projects/olir/informative-reference-catalog/details?referenceId=99#/) or [scf-olir-strm-nist-csf-2-0.xlsx](https://csrc.nist.gov/projects/olir/informative-reference-catalog/details?referenceId=153#/)

**Comment by tjx62** [comment]
Hi, is there any update on this feature's release?

**Comment by hejjoe** [comment]
the german bsi is using oscal for the [upcoming version of grundschutz++](https://github.com/BSI-Bund/Stand-der-Technik-Bibliothek/tree/main/Kompendien/Grundschutz%2B%2B-Kompendium). so supporting oscal would be helpful.
